addons/survey/static/src/scss/survey_form.scss

```diff
@@ -14,6 +14,10 @@ nav#oe_main_menu_navbar {
 /**********************************************************
                         Common Style
  **********************************************************/
+div#wrapwrap {
+    transition: box-shadow 0.3s ease-in-out;
+}
+
```

Comment:
Ouch.

Can we add the in the survey_form widget?

Like at the start, add a class on the wrapwrap, something like "o_survey_wrapwrap"?
Adding rules on global elements is always a bit scary.